ui/base-tool/src/main/res/values/styles.xml

```diff
@@ -5,9 +5,10 @@
         <item name="toolbarStyle">@style/Widget.GodTools.Tool.Toolbar.AppBar</item>
     </style>
 
-    <style name="Theme.GodTools.Tool.AppBar">
+    <style name="Theme.GodTools.Tool.AppBar" parent="Theme.AppCompat.NoActionBar">
```

Comment:
don't include the parent attribute, it actually breaks how we are inheriting style data

Reply:
without the parent attribute it automatically inherits from "Theme.GodTools.Tool", which is the expected behavior.